libs/newsletter/src/models/newsletter.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface DaffNewsletter {
```

Comment:
Probably should be called `DaffNewsletterSubmission`

Reply:
Okay, this was updated through the library.